startup.py

```diff
@@ -24,8 +24,8 @@ class AliasLauncher(SoftwareLauncher):
     context.
     """
 
-    # The name used to establish the ShotGrid Alias Engine client with Alias
-    TK_ALIAS_CLIENT_NAME = "shotgrid"
+    # The name used to establish the Flow Production Tracking Alias Engine client with Alias
+    TK_ALIAS_CLIENT_NAME = "flow production tracking"
```

Comment:
We can't have spaces in the client name
```suggestion
    TK_ALIAS_CLIENT_NAME = "flow-production-tracking"
```